packages/extension-sdk-react/package.json

```diff
@@ -44,8 +44,10 @@
     "@looker/extension-sdk": "^21.4.1",
     "@looker/sdk": "^21.4.1",
     "@looker/sdk-rtl": "^21.0.12",
+    "lodash": "^4.17.20"
+  },
+  "peerDependencies": {
     "history": "^4.9.0",
-    "lodash": "^4.17.20",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
```

Comment:
Would it be possible to require a slightly lower minimum version? In components we do:
```
    "react": "^16.11",
    "react-dom": "^16.11",
```

Reply:
Should we downgrade all our other React packages in this repository to these versions?

Reply:
No, I don't think so. In components we actually put a [higher version in `devDependencies`](https://github.com/looker-open-source/components/blob/main/packages/components/package.json#L58) just to stay as up-to-date as possible. But having a lower version in `peerDependencies` just means that as long as the developer installs that version or higher alongside this package, they're good to go. (And they won't get a pesky duplicate React warning.)

Reply:
Thanks for explaining that!

Reply:
So is it okay to have a different version in dev dependencies for this eaxmple?